examples/tutorials/mocking.md

```diff
@@ -351,11 +351,11 @@ class UserService {
   }
 }
 
-Deno.test("partial mocking with spies", async () => {
+Deno.test("partial mocking with stubs", async () => {
   const service = new UserService();
 
   // Only mock the getUser method
-  const getUserSpy = spy(
+  const getUserMock = stub(
```

Comment:
`spy` doesn't have capability of replacing the implementation. We need to use `stub` to replace it